Add anatomy diagrams and remaining docs for ARIA color components (#1603)
adobe/react-spectrum: packages/@react-aria/color/docs/useColorSlider.mdx

### Vertical

This example shows how to build a vertical color slider. The main difference from horizontal color sliders is the addition of the
`orientation: 'vertical'` option to `useColorSlider`. This automatically adjusts the internal positioning and dragging logic.

> Additionally, the vertical slider does not have a visible label or `<output>` element. This is done by simply not using the returned

Reviewer: ```suggestion
Additionally, this example does not have a visible label or `<output>` element. This can be done by simply not using the returned
```

Reviewer: Slider has "Additionally, this example shows how to build a slider without a visible label or output element."

### HSLA

> This example shows how to build a similar color picker to the one above, but for HSLA colors.

Reviewer: ```suggestion
This example shows how to build a similar color picker to the one above, using HSLA colors instead.
```
